test/media.test.js

```diff
@@ -64,6 +64,13 @@ describe("Image", () => {
       img.src = URL
     })
 
+    test("set size vs natural size", () => {
```

Comment:
Should this be in here? This whole commit, actually...?

Reply:
You're right. I moved those into their own section so that one's redundant now...